scripts/collect_workloads.py

```
@@ -0,0 +1,1231 @@
+def _tp_ep_from_definitions(def_files: list[Path]) -> tuple[int, int]:
+    """Extract tp and ep values from definition tags.
+
+    Reads all definition JSONs, collects tp:N and ep:N tags, verifies they
+    are consistent across definitions, and returns (tp, ep). Defaults to (1, 1).
+    """
+    tp_values, ep_values = set(), set()
+    for f in def_files:
+        try:
+            defn = json.loads(f.read_text())
+        except Exception:
+            continue
```

Ruff (0.15.7)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)